starter-code/scripts/articleView.js

```diff
@@ -75,35 +75,58 @@ articleView.setTeasers = function() {
 
 articleView.initNewArticlePage = function() {
   // TODO: Ensure the main .tab-content area is revealed. We might add more tabs later or otherwise edit the tab navigation.
+  $('.main-nav').on('click', '.tab', function() {
+    $('.tab-content').hide();
+    $('#' + $(this).data('content')).fadeIn();
+  });
 
+  $('.main-nav .tab:first').click();
+  $('.tab-content').show();
 
   // TODO: The new articles we create will be copy/pasted into our source data file.
   // Set up this "export" functionality. We can hide it for now, and show it once we have data to export.
+  $('#article-export').hide();
+
 
   $('#article-json').on('focus', function(){
     this.select();
   });
 
   // TODO: Add an event handler to update the preview and the export field if any inputs change.
-
+  $('form').on("change", function() {
+    articleView.create();
+    console.log("form was changed");
+  })
 };
 
 articleView.create = function() {
   // TODO: Set up a var to hold the new article we are creating.
   // Clear out the #articles element, so we can put in the updated preview
-
+  var newRawData = {};
+  newRawData.title = $('#title').val();
+  newRawData.body = $('#body').val();
+  newRawData.author = $('#authorName').val();
+  newRawData.authorUrl = $('#authorUrl').val();
+  newRawData.category = $('#category').val();
+  newRawData.publishedOn = $('#published').prop("clicked") ? new Date(): "";
+  $('#article-export').empty();
 
   // TODO: Instantiate an article based on what's in the form fields:
-
+  var newArticle = new Article(newRawData);
+  console.log(newArticle);
 
   // TODO: Use our interface to the Handblebars template to put this new article into the DOM:
-
+  console.log(newArticle.toHtml())
+  $('#article-export').append(newArticle.toHtml());
```

Comment:
should be appended to `#preview`
`#article-export` is for the data copy/paste into blogArticles.js